Source/Core/Core/PowerPC/Jit64/rangeset.h

```diff
@@ -0,0 +1,986 @@
+ /*
+  *  This file is part of libcxxsupport.
+  *
+  *  libcxxsupport is free software; you can redistribute it and/or modify
+  *  it under the terms of the GNU General Public License as published by
+  *  the Free Software Foundation; either version 2 of the License, or
+  *  (at your option) any later version.
+  *
+  *  libcxxsupport is distributed in the hope that it will be useful,
+  *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  *  GNU General Public License for more details.
+  *
+  *  You should have received a copy of the GNU General Public License
+  *  along with libcxxsupport; if not, write to the Free Software
+  *  Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+  */
+ 
+ /*
+  *  libcxxsupport is being developed at the Max-Planck-Institut fuer Astrophysik
+  *  and financially supported by the Deutsches Zentrum fuer Luft- und Raumfahrt
+  *  (DLR).
+  */
+ 
+ /*! \file rangeset.h
+  *  Class for storing sets of ranges of integer numbers
+  *
+  *  Copyright (C) 2011-2017 Max-Planck-Society
+  *  \author Martin Reinecke
+  */
+ 
+ #ifndef PLANCK_RANGESET_H
+ #define PLANCK_RANGESET_H
+ 
+ #include <algorithm>
+ #include <vector>
+ #include <utility>
+ #include <iostream>
```

Comment:
This should be swapped out with `<ostream>` or any other stream header. `<iostream>` injects a static constructor into every translation unit that includes it, so this has a runtime cost that can be avoided.